src/components/icon/icon.tsx

```diff
@@ -0,0 +1,55 @@
+import { Component, Prop, Host, h } from '@stencil/core';
+import classNames from 'classnames';
+
+@Component({
+	tag: 'aui-icon',
+})
+export class Icon {
+	/** The icon name (eg: ) */
+	@Prop() name: string;
+	/** ARIA Label */
+	@Prop() ariaLabel?: string;
+	/** Branding */
+	@Prop() branding?: string;
```

Comment:
Dit is misschien niet de best gekozen naam. Waarom niet `className` conform de andere Antwerp UI libs?

Reply:
Ik ben hier voor dezelfde prop gegaan als de andere web components

Reply:
I see. Ik denk er nog even over na.